### ISSUE TITLE
Add tags or subcategories to store Details

### ISSUE THREAD

**Issue by elcreativo** [tastaturberuf-contao-modules/anyStores#13]
Damit würde sich die Liste im Frontend leichter filtern lassen. 
Optional sogar kompatibel mit dem JS Plugin Isotope http://isotope.metafizzy.co

Anwendungsfall wäre z.b. das Hinzufügen eines Gewerkes oder der Branche

**Comment by Tastaturberuf** [comment]
Wie würde man das sinnvoll umsetzen können? Welche Schritte wären nötig?

**Comment by kreativlokal** [comment]
Ich würde die Unterkategorien auch benötigen. Beispiel Sportverein mit mehreren Sportarten.
Ich lege die Kategorie Mitglieder an und kann direkt Unterkategorien vergeben:
![bildschirmfoto 2015-05-17 um 13 31 21](https://cloud.githubusercontent.com/assets/12482281/7670161/3c8a9bfc-fc9a-11e4-919d-0c29b1a006b7.png)

Wenn ich dann ein neues Mitglied (Neuer Standort) anlege, kann ich direkt mit Checkboxen die Unterkategorien/Sportarten auswählen, die zutreffen.

Bei der Suche müssten die Checkboxen auch angezeigt werden, um zu filtern.

In der Liste/Karte/Detailseite sollten die Unterkategorien als Tags/Lables o.ä. erscheinen.

**Comment by elcreativo** [comment]
Die Idee finde ich gut! 
Lässt sich das so umsetzen?

**Comment by Tastaturberuf** [comment]
Ja das sollte gehen. Ist halt aufwendig.

**Comment by elcreativo** [comment]
Was würdest du schätzen, wie hoch wäre der Aufwand bei so einer Erweiterung?

**Comment by Tastaturberuf** [comment]
Man muss im Detail schauen wie man das implementiert, weil die MultiColumnWizard-Abhänhigkeit soll verschwinden, ich bin damit nicht zufrieden. Der ListWizard sollte in diesem Fall ausreichen, eine Beschreibung der Kategorie ist eigentlich nicht nötig. Dann muss man sehen das alles optional bleibt. Außerdem könnte man die Frage stellen ob die jetzigen Kategorien dann überhaupt noch nötig sind.
Das sollten schon so 20+ Stunden sein.

**Comment by kreativlokal** [comment]
Stimmt, die Beschreibung ist nicht zwingend notwendig - der Titel sollte selbsterklärend sein.
Auf die jetzigen Kategorien kann man dann natürlich verzichten, wenn man Mitglieder, Stores, etc. mehreren Kategorien zuordnen kann.

Im ersten Step würde mir auch reichen, wenn ich die Kategorien im Backend anlegen und zuordnen kann und man dann im Frontend danach filtern kann.

**Comment by Tastaturberuf** [comment]
Ich hab mittlerweile eher ein Tag-System im Kopf:
- Die Tags werden vorher angelegt.
- Die Stores können beliebig (auch mehrfach) getagt werden (Aus dem Pool der angelegten Tags)
- Die Tags können bei der Suche berücksichtigt werden (Sowohl als AND oder OR, das müsste man im Suchmodul einstellen können)
- Die jetzigen Kategorien würde ich behalten und zusätzlich strukturieren zu können und auch für Mehrfach-Installationen.

Es gibt keinen Zeitplan dafür. Ich versuche an dieser Stelle mal die aktuellen Gedanken festzuhalten.
